### ISSUE TITLE
Incorect Cached memory calculation

### ISSUE THREAD

**Issue by starinacool** [munin-monitoring/munin#1591]
We should not decrease cached memory by amount of shared memory.
We should exclude shared memory from apps memory.

https://github.com/munin-monitoring/munin/blob/16eff1e959e1ed3dbdb615c3834a2a596efebcc9/plugins/node.d.linux/memory#L304C80-L304C80

**Comment by steveschnepp** [comment]
Thanks!

Do you have some documentation that shows our implementation is incorrect ?

**Comment by steveschnepp** [comment]
You already replied in https://github.com/munin-monitoring/munin/pull/1592#issuecomment-1801190601 👍

**Comment by BETLOG** [comment]
I was about to ask a similar question. Is this issue whats causing this slightly horrifying graph escalation?
![2023-11-19--14-08-10_betlogbeast_memoryMuninmuninbetlogbeastmemoryGoogleChrome](https://github.com/munin-monitoring/munin/assets/12265451/7162fe4a-8e4d-40c7-a14f-d71b1e19534d)

**Comment by steveschnepp** [comment]
No. that is `CommittedAS` which is increasing too much. And it seems to be due to some change between the kernel version 5.10 and 6.1, as a debian 11 to 12 upgrade shows it.
